packages/compass-e2e-tests/helpers/commands/wait-for-aria-disabled.ts

```diff
@@ -3,20 +3,18 @@ import type { CompassBrowser } from '../compass-browser';
 
 export async function waitForAriaDisabled(
   browser: CompassBrowser,
-  selector:
-    | string
-    | ChainablePromiseElement<WebdriverIO.Element>
-    | WebdriverIO.Element,
```

Comment:
No need to also allow WebdriverIO.Element for convenience because you can't ordinarily get that type using .$() lookups.